domains/_vercel.nyll.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "NyllRE",
+        "email": "52743522+NyllRE@users.noreply.github.com"

```

Invalid email, you can safely remove this if you dont want to share it
```suggestion
        "username": "NyllRE"
```

---

is using my main email required? I'm afraid it would be an easy way for spammers to obtain public emails through the repository. I would happily provide it directly to someone on the team privately.